gen-api-reference/README.md

```diff
@@ -38,3 +38,40 @@ at the root of the repo, and that your GOPATH is set properly. For example:
 1. Run `git diff` to verify the changes.
 
 1. Create a PR to merge your changes.
+
+# Generating the Pipelines API Reference Page
+
+These are instructions for generating a copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/) page from the swagger file. 
```

Comment:
Suggestion: Revise to be prescriptive ("Use this to" instead of "this is a"), for example:

Use these instructions to generate an updated copy of the [Pipelines API Reference](https://www.kubeflow.org/docs/pipelines/reference/api/kubeflow-pipeline-api-spec/).

Reply:
Agreed, updated.